app/components/TipsDetails.tsx

```diff
@@ -72,17 +76,20 @@ export const TipsDetails = ({ schoolName, canEdit }: { schoolName: string; canEd
     }
   };
 
-  // Render the delete action
+  // Render the delete action if the user is an admin
   const renderRightActions = (tipID: string) => {
-    return (
-      <Pressable style={style.deleteButton} onPress={() => handleDelete(tipID)}>
-        <Text style={style.deleteButtonText}>Delete</Text>
-      </Pressable>
-    );
+    if (userInfo?.isAdmin) {
```

Comment:
Might be cleaner to put the isAdmin check directly in the JSX, so if isAdmin is not true, then we render the inner children without the Swipeable. I don't know if there is a isEnabled prop in Swipeable already, we could pass it in if that prop exists.